pic-sure-hpds-ui/src/main/webapp/picsureui/header/header.js

```diff
@@ -65,6 +65,9 @@ define(["jquery", "backbone","handlebars", "text!header/header.hbs", "overrides/
         		profileOverride.userProfile(event, this);
         	} else {
 	            userFunctions.meWithToken(this, function(user){
+	                if ($("#modal-window").length === 0) {
+	                    $('#main-content').append('<div id="modal-window"></div>');
```

Comment:
Add to main-content so it gets destroyed on page changes. This is to avoid duplicate ids for pages that include a modal-window element.